app/core/api/serializers/user.py

```diff
@@ -7,3 +7,13 @@ class PersonSerializer(serializers.ModelSerializer):
     class Meta:
         model = Person
         fields = ['id', 'username', 'avatar', 'full_name', 'is_active']
+
+class UserDetailSerializer(serializers.ModelSerializer):
```

Comment:
Use one serializer only.

Reply:
Also, the user should add manual fields such as first_name, father_name, etc. There are some fields like id, created_date, and updated_date that are set automatically.